core/templates/pages/skill-editor-page/modal-templates/skill-editor-save-modal.component.ts

```diff
@@ -28,7 +28,15 @@ import { AppConstants } from 'app.constants';
 export class SkillEditorSaveModalComponent extends ConfirmOrCancelModal {
   commitMessage: string;
   MAX_COMMIT_MESSAGE_LENGTH: number = AppConstants.MAX_COMMIT_MESSAGE_LENGTH;
-  constructor(activeModal: NgbActiveModal) {
+
+  constructor(
+    private activeModal: NgbActiveModal,
+    private changeDetectorRef: ChangeDetectorRef,
+  ) {
     super(activeModal);
   }
+
+  updateSaveDraftButton(): void {
+    this.changeDetectorRef.detectChanges();
+  }
```

Comment:
Is this necessary?

Reply:
yes e2e are failing without this after this component is migrated to angular